SCClassLibrary/Common/GUI/PlusGUI/Control/ServerPlusGUI.sc

```diff
@@ -1,33 +1,42 @@
 ServerTreeView {
 	var <server, <bounds, viewParent, <window, resp, updateFunc, updater, tabSize = 25, treeViewStatus;
 
-	*new { |server, bounds=(Rect(128, 64, 400, 400)), parent|
-		if(bounds.asRect.width < 400) {
-			var bnds = bounds.asRect;
-			"The width value you set will be changed to 400, the minimum width.".postln;
-			bounds = Rect(bnds.left, bnds.top, 400, bnds.height);
-		};
+	*new { |server, bounds, parent|
+		server = server ? Server.default;
```

Comment:
`server  = server ?? { Server.default };` would be optimised.

Reply:
done!